addons/test_mass_mailing/tests/test_performance.py

```diff
@@ -30,7 +30,7 @@ def test_send_mailing(self):
             'mailing_domain': [('id', 'in', self.mm_recs.ids)],
         })
 
-        with self.assertQueryCount(__system__=2432, marketing=3088):
+        with self.assertQueryCount(__system__=2433, marketing=3089):
```

Comment:
I guess this comes from the new stored field of the mixin ?

Reply:
I guess too :D